Source/Shaders/SkyAtmosphereFS.glsl

```diff
@@ -32,15 +32,38 @@
  
  // Code:  http://sponeil.net/
  // GPU Gems 2 Article:  http://http.developer.nvidia.com/GPUGems2/gpugems2_chapter16.html
- 
+ // HSV <-> RGB conversion with minimal branching: http://lolengine.net/blog/2013/07/27/rgb-to-hsv-in-glsl
+
+uniform vec3 u_hsvShift; // hue, saturation, value
+
 const float g = -0.95;
 const float g2 = g * g;
+const float epsilon = 1.0e-10;
+const vec4 K_RGB2HSB = vec4(0.0, -1.0 / 3.0, 2.0 / 3.0, -1.0);
+const vec4 K_HSB2RGB = vec4(1.0, 2.0 / 3.0, 1.0 / 3.0, 3.0);
+
 
 varying vec3 v_rayleighColor;
 varying vec3 v_mieColor;
 varying vec3 v_toCamera;
 varying vec3 v_positionEC;
 
+vec3 rgb2hsb(vec3 rgbColor)
```

Comment:
You could also experiment with the `czm_hue` and `czm_saturation` built-in functions.